services/viva-ms/serverless.yml

```diff
@@ -2,7 +2,7 @@ service: viva-ms
 
 plugins:
   - serverless-bundle
-  - serverless-offline
```

Comment:
Don't we need the offline plugin in this ms?

Reply:
No. Can't run event offline?

Reply:
But you can invoke the function if you would like to?

Reply:
Hmm...